python/ray/tune/api_v2/convertible_to_trainable.py

```diff
@@ -0,0 +1,10 @@
+# ONLY PLACEHOLDER TO BE REMOVED!!
+import abc
+from typing import Type
+
+from ray.tune import Trainable
+
+
+class ConvertibleToTrainable(abc.ABC):
```

Comment:
(PublicAPI annotation)

Reply:
This is included to Amog's PR. So to be removed actually...